materials/tasks.py

```diff
@@ -0,0 +1,19 @@
+from celery import shared_task
+from django.core.mail import send_mail
+from config.settings import EMAIL_HOST_USER
+from users.models import Subscription, User
+
+
+@shared_task
+def sendmail_course_updated(course):
+    '''Отправляет уведомление о том, что курс обновлен'''
+    subscription_course = Subscription.objects.filter(course=course)
```

Comment:
А тут искать по id курса
```suggestion
    subscription_course = Subscription.objects.filter(course_id=course_id)
```